atest/testdata/standard_libraries/builtin/variables_to_verify.py

```diff
@@ -1,6 +1,6 @@
 import os
 
-from robot.utils import OrderedDict
+from collections import OrderedDict
```

Comment:
nit-pick:  [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) states

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
> 
> Imports should be grouped in the following order:
> 
>   1. standard library imports
>   1. related third party imports
>   1. local application/library specific imports
> 
> You should put a blank line between each group of imports.

As a result, can you re-organize your import statements?

Reply:
@bkhouri I'll update it. Within each group, which order is preferred?
```python
from X import X
import X
```
```python
import X
from X import X
```

Reply:
I think think there is strict guidelines with respect to "how to do the import".  The PEP8 section on imports can be found here https://www.python.org/dev/peps/pep-0008/#imports

Reply:
No mention in PEP8. Google recommend this:
```python
import foo
from foo import bar
from foo.bar import baz
from foo.bar import Quux
from Foob import ar
```
https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting

I've used [isort](https://github.com/timothycrosley/isort) to update these files like this.